.maestro/setup_ui_tests.sh

```diff
@@ -4,17 +4,6 @@
 
 source $(dirname $0)/common.sh
 
-## Constants
```

Comment:
Did you mean to delete all this or was it temporary?  Likewise for the build function below.